src/Components/Header/Header.css

```diff
@@ -97,9 +105,10 @@
 }
 
 @keyframes bell {
```

Comment:
I've used keyframes in the past because of....chatgpt haha, but what actually is it doing? I'll ask this in person too!